submissions/omolayoOdupe/todo_app/src/components/TodoContainer.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import InputTodo from './InputTodo';
+import TodoList from './TodoList';
+import TodoStatus from './TodoStatus'
+import './Todo.css'
+
+class TodoContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { 
+            items:[],
+            currentItem: {text:'', key:'', completed:false}    
+         }
+    }
+
+    inputTodoHandler = (e) =>{
+        this.setState({
+            currentItem: { 
+                text: e.target.value, 
+                key:Date.now()
+            }
+        })
+    }
+
+    addItemHandler = (e) => {
+        e.preventDefault();
+
+        const newItem = this.state.currentItem;
+
+        if (newItem !==''){
+            
+            const newtodo=[...this.state.items, newItem];
+
+            this.setState({
+                items: newtodo,
+                currentItem: {
+                    text: '',
+                    key:'',
+                    completed:false
+                }   
+            })    
+        }
+    }
+
+    toogleCheckBox = (key) => {
+        this.setState({
+            items:this.state.items.filter(item=>{
+                if (item.key === key) {
+                    item.completed = !item.completed;
+                }
+                return item;
+            })
+        })
+    }
+
+    itemsLeft = () => {
+        return this.state.items.filter((item) => !item.completed);
+    }
+
+    render() { 
+        return (
+            <div className='all-content'>
+                <div className='head-box'>
+                </div>
```

Comment:
There is a closing div here. I think you should check it the structure again @OmolayoO